### PR TITLE
Remove doctrine command from install process

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Remove doctrine command from install process
| Type?             | improvement
| Category?         | IN
| BC breaks?        | yes
| Deprecations?     | no
| How to test?      | CI green and UI tests green
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/9641108642
| Fixed issue or discussion?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### Breaking changes
- The command `prestashop:schema:update-without-foreign` is no longer used in the install process
- The command `prestashop:schema:update-without-foreign` now has two optional parameter `--dump-sql` and `--force` The default behaviour is NOT to apply changes and NOT to output anything, so the default behaviour does nothing now
- The command was tracking module entities (in particular ps_comments) and was updating the DB of those entities, since it's not called anymore the modules are fully responsible for their DB structure now

### CODE REVIEW

File: install-dev/data/db_structure.sql [hunk 1]
```diff
@@ -829,7 +829,8 @@ CREATE TABLE `PREFIX_employee` (
   PRIMARY KEY (`id_employee`),
   KEY `employee_login` (`email`, `passwd`),
   KEY `id_employee_passwd` (`id_employee`, `passwd`),
-  KEY `id_profile` (`id_profile`)
+  KEY `id_profile` (`id_profile`),
+  KEY `IDX_1D8DF9EBBA299860` (`id_lang`)
```
Comment (matthieu-rolland) [inline install-dev/data/db_structure.sql:833]: do you know why this index is removed ?

Comment (jolelievre) [reply install-dev/data/db_structure.sql:833]: It's not removed, it's added The initial table creation only created one And Doctrine systematically added this extra one via the command

**Review by M0rgan01 — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Only automated tests are required to validate this PR, no need for manual testing.

**Comment by ps-jarvis** [comment]
QA approved, well done! Message to the maintainers: do not forget to milestone it before the merge.

**Pull request merged**